packages/arb-token-bridge-ui/tests/e2e/specs/withdrawERC20.cy.ts

```diff
@@ -114,6 +129,49 @@ describe('Withdraw ERC20 Token', () => {
       })
     })
 
+    it('should claim funds', { defaultCommandTimeout: 200_000 }, () => {
+      // increase the timeout for this test as claim button can take ~(20 blocks *10 blocks/sec) to activate
+
+      cy.login({ networkType: 'parentChain' }) // login to L1 to claim the funds (otherwise would need to change network after clicking on claim)
+
+      cy.findByLabelText('Open Transaction History')
+        .should('be.visible')
+        .click()
+
+      cy.findClaimButton(
+        formatAmount(ERC20AmountToSend, {
+          symbol: 'WETH'
+        })
+      )
+        .click()
+        .then(() => {
```

Comment:
we don't need `then` here

Reply:
Removed all then's added in the current diff.